packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1283,19 +1283,14 @@ class BuildOwner {
         int dirtyCount = _dirtyElements.length;
         int index = 0;
         while (index < dirtyCount) {
-          assert(() {
```

Comment:
Assuming that you meant to remove this assert, should we also remove the definition of debugPrintRebuildDirtyWidgets?